src/main/java/cm/ptks/craftflowers/storage/LanguageFile.java

```diff
@@ -0,0 +1,462 @@
+  public static String FLOWER_HORN_CORAL;
+  public static String FLOWER_TUBE_CORAL;
+  public static String FLOWER_BRAIN_CORAL_FAN;
+  public static String FLOWER_BUBBLE_CORAL_FAN;
+  public static String FLOWER_FIRE_CORAL_FAN;
+  public static String FLOWER_HORN_CORAL_FAN;
+  public static String FLOWER_TUBE_CORAL_FAN;
+
+  public static String FLOWER_WHEAT;
+  public static String FLOWER_WHEAT_AGE_0;
+  public static String FLOWER_WHEAT_AGE_1;
+  public static String FLOWER_WHEAT_AGE_2;
+  public static String FLOWER_WHEAT_AGE_3;
+  public static String FLOWER_WHEAT_AGE_4;
+  public static String FLOWER_WHEAT_AGE_5;
+  public static String FLOWER_WHEAT_AGE_6;
+  public static String FLOWER_WHEAT_AGE_7;
+
+  public static String FLOWER_POTATO;
+  public static String FLOWER_POTATO_AGE_0;
+  public static String FLOWER_POTATO_AGE_1;
+  public static String FLOWER_POTATO_AGE_2;
+  public static String FLOWER_POTATO_AGE_3;
+  public static String FLOWER_POTATO_AGE_4;
+  public static String FLOWER_POTATO_AGE_5;
+  public static String FLOWER_POTATO_AGE_6;
+  public static String FLOWER_POTATO_AGE_7;
+
+  public static String FLOWER_CARROT;
+  public static String FLOWER_CARROT_AGE_0;
+  public static String FLOWER_CARROT_AGE_1;
+  public static String FLOWER_CARROT_AGE_2;
+  public static String FLOWER_CARROT_AGE_3;
+  public static String FLOWER_CARROT_AGE_4;
+  public static String FLOWER_CARROT_AGE_5;
+  public static String FLOWER_CARROT_AGE_6;
+  public static String FLOWER_CARROT_AGE_7;
+
+  public static String FLOWER_AIR;
+
+  public static String FLOWER_POT;
+
+  public static String GUI_TITLE;
+  public static String GUI_CREATE_FLOWER;
+  public static String GUI_CANT_CREATE_EMPTY_FLOWER;
+  public static String GUI_REMOVE_FROM_TOP;
+  public static String GUI_CLEAR_ALL;
+  public static String GUI_FLOWER_GROUP_TITLE_COLOR;
+  public static String GUI_GLASS_NAME_COLOR;
+  public static String GUI_PREVIOUS;
+  public static String GUI_NEXT;
+  public static String GUI_SAVED_FLOWERPOTS;
+  public static String GUI_SAVED_FLOWERPOTS_NAME_COLOR;
+
+  public static String ACTION_NO_PERMISSION_PLACE;
+  public static String ACTION_MISSING_FOLLOWING_ITEMS;
+  public static String ACTION_MISSING_FOLLOWING_ITEMS_LIST;
+  public static String ACTION_NO_PERMISSION_EDIT;
+  public static String ACTION_OUTDATE_VERSION;
+  public static String ACTION_CURRENT_VERSION;
+  public static String ACTION_NEWEST_VERSION;
+  public static String ACTION_DOWNLOAD_NEW_VERSION;
+  public static String ACTION_FAILED_CHECK_VERSION;
+
+  public static void LoadLanguageFile() {
+    CraftFlowers instance = CraftFlowers.getInstance();
+
+    String path = "language.yml";
+
+    File languageFile = new File(instance.getDataFolder(), path);
+
+    if (!languageFile.exists()) {
+      instance.saveResource(path, false);
+    }
+
+    YamlConfiguration defaultLanguage =
+        YamlConfiguration.loadConfiguration(new InputStreamReader(instance.getResource(path)));
```

Comment:
Do new InputStreamReader(stream, Charsets.UTF_8) here. This should fix your issue